README.md

```diff
@@ -49,6 +49,32 @@ To deploy a Opennebula instance locally run:
 
 where "boxname" can be debian or centos
 
+## Docker
+
+To deploy a Opennebula instance locally in a docker container run these commandos.
+
+Build base image with puppet in it (Depending on centos:latest):
+
+    docker build --rm -f docker/Dockerfile-Centos6 -t epost-dev/puppet .
+
+Build a image with the one puppet module in it:
+
+    docker build --rm -f docker/Dockerfile -t epost-dev/one .
+
+Run puppet in the container, choose one:
+
+    docker run --rm epost-dev/one puppet apply /etc/puppet/modules/one/spec/docker-int/one-head.pp
+    docker run --rm epost-dev/one puppet apply /etc/puppet/modules/one/spec/docker-int/one-node.pp
+    docker run --rm epost-dev/one puppet apply /etc/puppet/modules/one/spec/docker-int/one-head-node.pp
+
+The files can be found in the spec directory of this project. One will build a one head,
+one will build a node and one a head which also can be a node.
+
```

Comment:
Better description needed:
the three commands rely upon the second build command.
Add details of applied manifests by specifying which manifest will deliver which functionality.